src/ui/public/agg_response/tabify/_buckets.js

```diff
@@ -81,4 +82,20 @@ TabifyBuckets.prototype._orderBucketsAccordingToParams = function (params) {
   }
 };
 
+TabifyBuckets.prototype._dropPartials = function (params, timeRange) {
+  if (params.drop_partials && !this.objectMode && this.buckets.length > 1) {
```

Comment:
rather than wrapping the whole thing in an `if` statement return early:

`if (!params.drop_partials || this.objectMode || this.buckets.length <= 1) { return; }`

Reply:
maybe move the `if (params.drop_partials)` out of this function ?

also join with below if statement ... and extract to variables so its easier to read:
```
 const isTimeField = params.field.name === timeRange.name;
if (this.objectMode || this.buckets.length <= 1 || !isTimeField) { 
 return;
}
```